File: URI/2863.py
while(1):
    try:
        N = int(input())
        menor = 0
        for i in range(N):
            num = float(input())
            if i == 0:
                menor = num
            elif menor > num:
                menor = num
        print("{:.2f}".format(menor))
    except EOFError:
        break;

File: URI/C/2863.c
#include<stdio.h>

int main(){
    int N;
    double menor,num;
    while(scanf("%d",&N) != EOF){
        for(int i=0; i<N; i++){
            scanf("%lf",&num);
            if(i == 0)
                menor = num;
            else if(menor > num)
                menor = num;
        }
        printf("%.2lf\n",menor);
    }
    return 0;
}
